dashboard/modules/usage_stats/usage_stats_head.py

```diff
@@ -41,6 +51,61 @@ async def get_usage_stats_enabled(self, req) -> aiohttp.web.Response:
                 usage_stats_prompt_enabled=self.usage_stats_prompt_enabled,
             )
 
+    def _check_grafana_running(self):
+        from ray._private.usage.usage_lib import TagKey, record_extra_usage_tag
+
+        if self._grafana_ran_before:
+            return
+
+        resp = requests.get(f"{self._dashboard_url_base}/api/grafana_health")
+        grafana_running = False
+        try:
+            if resp.status_code == 200:
+                json = resp.json()
+                grafana_running = (
+                    json["result"] is True and json["data"]["grafanaHost"] != "DISABLED"
+                )
+        except Exception:
+            pass
+
+        record_extra_usage_tag(
+            TagKey.DASHBOARD_METRICS_GRAFANA_ENABLED,
+            str(grafana_running),
+        )
+
+        if grafana_running:
+            # Don't need to update the tag ever again
+            self._grafana_ran_before = True
+
+    def _check_prometheus_running(self):
+        from ray._private.usage.usage_lib import TagKey, record_extra_usage_tag
+
+        if self._prometheus_ran_before:
+            return
+
+        resp = requests.get(f"{self._dashboard_url_base}/api/prometheus_health")
+        prometheus_running = False
+        try:
+            if resp.status_code == 200:
+                json = resp.json()
+                prometheus_running = json["result"] is True
+        except Exception:
+            pass
+
+        record_extra_usage_tag(
```

Comment:
I think what @rkooo567 means is that we can just directly add it to the `extra_usage_tags` field that the `usage_stats_head` will report to the server. Instead of uploading to gcs first and then downloading it immediately afterwards. But I'm fine with the current approach for consistency.